Dockerfile

```diff
@@ -13,6 +13,7 @@ RUN mkdir -p plugin/nodecache
 RUN echo 'nodecache:nodecache' >> /coredns/plugin.cfg
 
 COPY *.go /coredns/plugin/nodecache/
+RUN go get github.com/coreos/go-iptables@f901d6c2a4f2a4df092b98c33366dfba1f93d7a0 github.com/vishvananda/netlink@f049be6f391489d3f374498fe0c8df8449258372
```

Comment:
Why use commit hash rather than tags?

Reply:
What a tag points to can change, go.mod  usually has a go.sum to ensure two builds build the same.. so a weak try for more reproducibility. It's not less readable though, go.mod/sum would be neater :/

Reply:
Well, usually tags are treated as immutable and they are definitely well more maintainable than commit hashes (with no corresponding tag comment or CI to check if it matches go.mod file in this repository), but fair enough :)